### PR TITLE
Frontend issue resolved 

### PR DESCRIPTION
**Pull request by sazedul-haque**
1. Subscription badge removed
2. Only keep action and properties that starts with tutor_option
3. Remove duplicate clear button
4. Moved convertToErrorMessage to shared directory

### CODE REVIEW

File: assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx [hunk 1]
```diff
@@ -39,12 +39,12 @@ const CourseListTable = ({ onSelectClick }: CourseListTableProps) => {
...
                 <div css={styles.bundleBadge}>{sprintf(__('%d Course Bundle', 'tutor'), item.total_course)}</div>
               )}
-              {item.plan_start_price && (
+              {/* {item.plan_start_price && (
                 <div css={styles.subscriptionBadge}>
                   <SVGIcon name="dollar-recurring" width={16} height={16} />
                   {__('Subscription', 'tutor')}
                 </div>
-              )}
+              )} */}
```
Comment (b-l-i-n-d) [inline assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx:47]: Please remove it if not needed.

File: assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx [hunk 2]
```diff
@@ -63,7 +63,9 @@ const CourseListTable = ({ onSelectClick }: CourseListTableProps) => {
             </div>
             <div css={styles.price} data-price>
               <Show when={item.is_purchasable} fallback={__('Free', 'tutor')}>
-                <Show
+                <span>{item.sale_price ? item.sale_price : item.regular_price}</span>
+                {item.sale_price && <span css={styles.discountPrice}>{item.regular_price}</span>}
+                {/* <Show
```
Comment (b-l-i-n-d) [inline assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx:68]: Please remove it if not needed.

**Review by b-l-i-n-d — APPROVED** [review]

File: assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx (hunk 1 above)
Comment (sazedul-haque) [reply assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx:47]: We will need this in future.

File: assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx (hunk 2 above)
Comment (sazedul-haque) [reply assets/react/v3/entries/pro/manual-enrollment/components/modals/CourseListModal/CourseListTable.tsx:68]: We will need this in future.

**Pull request merged**